bundle/language_en.properties

```diff
@@ -1839,9 +1956,9 @@ angal.xmpp.isnowoffline.txt
 angal.xmpp.isnowonline.txt                                                                             = is now online
 angal.xmpp.me.txt                                                                                      = me
 angal.xmpp.sendfile.txt                                                                                = Send File
-angal.xmpp.thefiletransferofbetweenyouandendedsuccesfully.fmt.msg                                      = the file transfer of {0} between you and {1} ended successfully
+angal.xmpp.thefiletransferofbetweenyouandendedsuccessfully.fmt.msg                                     = the file transfer of {0} between you and {1} ended successfully
```

Comment:
```suggestion
angal.xmpp.thefiletransferofbetweenyouandendedsuccessfully.fmt.msg                                     = The file transfer of {0} between you and {1} ended successfully,
```

Reply:
should be that one a comma or a full stop?

Reply:
good catch; should be a period.